test/optimizer/sampling_pushdown.test

```diff
@@ -0,0 +1,52 @@
+# name: test/optimizer/sampling_pushdown.test
```

Comment:
Can we add a `test_slow` where we perform this on a larger table, e.g. using TPC-H?